### PR TITLE
add unit test for command info

### PR DESCRIPTION
**Pull request by lzhan**

### CODE REVIEW

File: tests/cli/lib/app/commands/test-info1.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        libpath = require('path'),
+        cmdpath = libpath.join(__dirname, '../../../../../lib/app/commands/info.js'),
+
+        mockery = require('mockery'),
+        mockPath,
+        info;
+
+    suite.add(new Y.Test.Case({
+
+        name: 'info test cases',
```
Comment (drewfish) [inline tests/cli/lib/app/commands/test-info1.js]: This suite has the same name as the suite found in tests/cli/lib/app/commands/test-info.js.  I don't know if that'll cause a problem or not.

File: tests/cli/lib/app/commands/test-info1.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        setUp: function() {
+        },
+
+        tearDown: function() {
+            mockery.deregisterAll();
+            mockery.disable({
+                'useCleanCache': true,
+                'warnOnUnregistered': false,
+                'warnOnReplace': false
+            });
```
Comment (drewfish) [inline tests/cli/lib/app/commands/test-info1.js:47]: It's a little weird having cleanup in tearDown() but the initialization isn't being done in setUp().  I suggest moving those mockery.whatever() calls from the test itself into setUp().

**Comment by lzhan** [comment]
Thanks Drew. Will update as you suggested.

**Comment by drewfish** [comment]
+1

**Pull request merged**